### PR TITLE
Remove scalapb

### PR DESCRIPTION
**Pull request by rafaparadela**
This PR removes scalapb. I need to organize all the PRs related to protobuf.

### CODE REVIEW

File: src/test/scala/higherkindness/skeuomorph/instances.scala [hunk 1]
```diff
@@ -30,8 +30,8 @@ import mu.MuF
 import avro.AvroF
 import protobuf.{FieldF, ProtobufF}
 import qq.droste.Basis
-import scalapb.UnknownFieldSet
-import scalapb.descriptors.FileDescriptor
+//import scalapb.UnknownFieldSet
+//import scalapb.descriptors.FileDescriptor
```
Comment (juanpedromoreno) [inline src/test/scala/higherkindness/skeuomorph/instances.scala]: I guess we could remove these lines

File: src/test/scala/higherkindness/skeuomorph/protobuf/ProtobufSpec.scala [hunk 2]
```diff
@@ -20,7 +20,7 @@ import cats.Functor
 import com.google.protobuf.descriptor.FieldDescriptorProto.Type
 import com.google.protobuf.descriptor.FieldDescriptorProto.Type._
 import org.scalacheck.Prop
-import scalapb.descriptors._
+//import scalapb.descriptors._
```
Comment (juanpedromoreno) [inline src/test/scala/higherkindness/skeuomorph/protobuf/ProtobufSpec.scala]: And this one

File: src/main/scala/higherkindness/skeuomorph/protobuf/Native.scala [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+    tpe: NativeDescriptor,
+    position: Int,
+    options: List[NativeOption],
+    isRepeated: Boolean,
+    isMapField: Boolean)
+    extends NativeFieldF
+
+final case class NativeOneOfField(name: String, tpe: NativeOneOf) extends NativeFieldF
+
+sealed trait NativeDescriptor
```
[the remark this answers is not in the document]
Comment (rafaparadela) [reply src/main/scala/higherkindness/skeuomorph/protobuf/Native.scala]: That was my first approach actually, but when I tried to express the conversion `Coalgebra[ProtobufF, FileDescriptorProto]` I realized that `FileDescriptorProto` is not algebraic enough. I can try again.

File: src/main/scala/higherkindness/skeuomorph/protobuf/Optimize.scala [hunk 4]
```diff
@@ -1,51 +0,0 @@
...
- * limitations under the License.
- */
-
-package higherkindness.skeuomorph.protobuf
-
-import higherkindness.skeuomorph.protobuf.ProtobufF.{TMessage, TRepeated}
-import higherkindness.skeuomorph.protobuf.FieldF._
-import qq.droste.{scheme, Basis, Trans}
-
-object Optimize {
```
[the remark this answers is not in the document]
Comment (rafaparadela) [reply src/main/scala/higherkindness/skeuomorph/protobuf/Optimize.scala]: Nope. These optimizations added by @rlmark in this object were only applicable to `ProtobufF`, but are no longer necessary given that the conversion from DescriptorProto to NativeDescriptor now deal with this scenario, unlike the previous approach.

**Comment by rafaparadela** [comment]
Closed in favor of #66

File: src/test/scala/higherkindness/skeuomorph/protobuf/ProtobufSpec.scala (hunk 2 above)
Comment (rafaparadela) [reply src/test/scala/higherkindness/skeuomorph/protobuf/ProtobufSpec.scala]: Solved

File: src/test/scala/higherkindness/skeuomorph/instances.scala (hunk 1 above)
Comment (rafaparadela) [reply src/test/scala/higherkindness/skeuomorph/instances.scala]: Solved

**Pull request closed without merging**